docs/_blog/announcing-anchor-outputs-channels.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing Anchor Outputs Channels"
+description: "LDK v0.0.116 has added support for opening channels with anchor outputs."
+date: "2023-08-01"
+authors:
+  - Wilmer Paulino
+tags:
+  - Channels
+  - Anchor Outputs
+  - Fees
+---
+
+The `v0.0.116` release of LDK has added support for opening channels with anchor outputs! This has
+been a much requested feature by our users as it brings channels a few security and usability
+improvements.
+
+Lightning channels rely on pre-signed transactions that its participants can broadcast to the
+network if they wish to close the channel unilaterally, e.g., when their counterparty is offline.
+Before the anchor outputs feature, participants continually negotiated their commitment
+transaction’s fees based on the current blockspace demand to ensure a timely confirmation if they
```

Comment:
Maybe "via the BOLT2 `update_fee` mechanism` if users desire to know more about legacy mechanism.